test/common/network/matching/inputs_integration_test.cc

```diff
@@ -29,6 +31,23 @@ constexpr absl::string_view yaml = R"EOF(
             value: foo
 )EOF";
 
+constexpr absl::string_view yaml_filter_state = R"EOF(
+matcher_tree:
+  input:
+    name: input
+    typed_config:
+      "@type": type.googleapis.com/envoy.extensions.matching.common_inputs.network.v3.{}
```

Comment:
Can you hardcode `FilterStateInput` here? It's not worth parametrizing something with one possible value.

Reply:
done